tools/wake/main.cpp

```diff
@@ -268,6 +279,7 @@ void print_help(const char *argv0) {
     << "    --last     -l      See --last-used"                                            << std::endl
     << "    --last-used        Capture all jobs used by last build. Regardless of cache"   << std::endl
     << "    --last-executed    Capture all jobs executed by the last build. Skips cache"   << std::endl
+    << "    --history          Report the cmndline history of all wake commands recorded"  << std::endl
```

Comment:
```suggestion
    << "    --history          Report the command-line history of all wake commands"  << std::endl
```
To match the spelling for `--in`
We don't need to specify "recorded" in the same way we don't specify that for things like "--list-outputs"